src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+function printMessage(messageText, messageClass = '') {
+  const message = document.createElement('div');
+
+  message.classList.add('message', messageClass);
+  message.innerText = messageText;
+
+  document.body.append(message);
+}
+
+const promise1 = new Promise((resolve) => {
+  const logoButton = document.querySelector('.logo');
+
+  logoButton.addEventListener('click', () => {
+    printMessage('Promise was resolved!', 'message');
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  reject(setTimeout(() => {
+    printMessage('Promise was rejected!', 'error-message');
+  }, 3000));
+});
+
+promise1
+  .then(() => promise2)
+  .catch(() => {});
```

Comment:
```suggestion
promise2.catch((errorMessage) => {
```
Для чого тут запускати перший проміс? 🤔  
Тобі ж потрібен тільки реджект другого